src/main/java/hu/eszter/bokkon/service/Util.java

```diff
@@ -1,22 +1,50 @@
 package hu.eszter.bokkon.service;
 
-import hu.eszter.bokkon.model.animal.Animal;
+import hu.eszter.bokkon.model.animal.*;
 import hu.eszter.bokkon.model.participants.Farmer;
 
 import java.util.*;
 
 public class Util {
 
-    private static final String ANSI_BRIGHT_BLUE   = "\u001B[94m";
-    private static final String ANSI_BLUE   = "\u001B[34m";
-    private static final String ANSI_CYAN   = "\u001B[36m";
-    private static final String ANSI_BRIGHT_CYAN   = "\u001B[96m";
+    /**
+     * a list containing Animals is to be used as both horizontal and vertical indeces for the exchangeRates matrix,
+     * therefore their order is important
+     */
+    private static final List<Animal> animalIndecesERates = new ArrayList<>(Arrays.asList(Animal.RABBIT,
```

Comment:
you can exploit the enum's ordinal() method instead of maintaining and index-list